site/content/internal/release-process.md

```diff
@@ -43,4 +43,39 @@ After a couple days pass you will need to set the CI servers to point back to `m
 1. Post `/mb setci master` into a private channel.
 2. Post `/mb setprerelease master` into a private channel.
 
+# Daily Merges from Release Branch to Master Branch
+
+During the release/rc cuts we need to merge daily the release branch into master to sync all changes made in the release branch.
```

Comment:
Maybe we should add a note about asking core committers to hold off merging?

Reply:
sure thing! thanks for the review!

Reply:
done